### PR TITLE
fix: disable read receipts

### PR DESCRIPTION
**Pull request by alexandreferris**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Sending read receipt confirmation for messages are very slow.

### Causes (Optional)

Probably due to current performance issues and `STORAGE_ERROR` issue, although with upcoming improvements this issue won't be happening.

### Solutions

Disable for now until we find solutions.

### CODE REVIEW

**Review by vitorhugods — APPROVED** [review]
Probably some tests we need to `@Ignore` for now?

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt [hunk 1]
```diff
@@ -37,7 +37,8 @@ class UpdateConversationReadDateUseCase internal constructor(
      */
     suspend operator fun invoke(conversationId: QualifiedID, time: Instant) {
         selfConversationIdProvider().flatMap { selfConversationIds ->
-            sendConfirmation(conversationId)
+            // TODO: Disabled for now as we are still figuring out performance and STORAGE_ERROR issues.
+            // sendConfirmation(conversationId)
             conversationRepository.updateConversationReadDate(conversationId, time.toIsoDateTimeString())
             selfConversationIds.foldToEitherWhileRight(Unit) { selfConversationId, _ ->
                 sendLastReadMessageToOtherClients(conversationId, selfConversationId, time)
```
Comment (mchenani) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt:44]: how about disabling the sendReadReceipt to the self aswell?

**Review by yamilmedina — APPROVED** [review]
🧑‍🚒

**Review by mchenani — APPROVED** [review]
one question...

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt (hunk 1 above)
Comment (alexandreferris) [reply logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt:44]: As they are two different things I suppose we still want to keep updating the last time we read the conversation? 🤔

Comment (mchenani) [reply logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt:44]: so still we're sending messages related to the read-receipts [one less in this case :D ] but   still sending! so I my suggestion to cut all the connections with outside related to read-receipts.

Comment (tmspzz) [reply logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt:44]: we want to update the last_read_date locally but don't send anything

**Pull request merged**